backend/api/Profiles/ParcelProfile.cs

```diff
@@ -12,21 +12,23 @@ public class ParcelProfile : Profile
         #region Constructors
         public ParcelProfile()
         {
-            CreateMap<Entity.Parcel, Models.Parts.ParcelModel>();
+            CreateMap<Entity.Parcel, Models.Parts.ParcelModel>()
+                .ForMember(dest => dest.PID, opt => opt.MapFrom(src => src.ParcelIdentity));
```

Comment:
Does not impact this PR but some comments/documentation on what makes a `Models.Parts.ParcelModel` different from a `Models.ParcelModel` would go a long way in understanding the naming on the API models

Reply:
Yeah, I actually am not clear on that either.

Reply:
We should come up with a better way to name them.  Models are often specific to endpoints (although they are often shared by many).

Some models we only want to return specific data.  We don't want to send all information in every endpoint request.

If I make a request for a list of Parcels and I only need to display two of their properties, I shouldn't be receiving all of their properties in the request.  While simpler to understand it doesn't consider network performance and costs associated with the extra data over time.